build.sbt

```diff
@@ -701,16 +751,18 @@ lazy val scalaDist = Project("scala-dist", file(".") / "target" / "scala-dist-di
   )
   .dependsOn(libraryAll, compiler, scalap)
 
-lazy val root = (project in file("."))
+lazy val root: Project = (project in file("."))
   .settings(disableDocs: _*)
   .settings(disablePublishing: _*)
   .settings(generateBuildCharacterFileSettings: _*)
   .settings(
     publish := {},
     publishLocal := {},
     commands ++= ScriptCommands.all,
-    Quiet.silenceIvyUpdateInfoLogging
-)
+    Quiet.silenceIvyUpdateInfoLogging,
+    antStyle := false,
+    incOptions := incOptions.value.withNameHashing(!antStyle.value).withAntStyle(antStyle.value)
```

Comment:
I've asked the sbt guys for help with this because I couldn't make it work. If there's no cleaner solution, I could move the definition into an object (which can be referenced from all `.sbt` files) and add an alias in `build.sbt` to make it available with an unqualified name on the command line.